server/route.go

```diff
@@ -362,6 +362,10 @@ func (c *client) processRouteInfo(info *Info) {
 	s := c.srv
 	remoteID := c.route.remoteID
 
+	c.mu.Unlock()
+	gacc := c.srv.globalAccount()
```

Comment:
Code is still in this PR, did you forget to commit/push force?

Reply:
I squashed it before, but removed now.